hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -12,7 +12,9 @@
 from hmalib import metrics
 from hmalib.common.logging import get_logger
 from hmalib.common.s3_adapters import ThreatExchangeS3PDQAdapter, S3ThreatDataConfig
-from hmalib.models import PDQMatchRecord, PipelinePDQHashRecord
```

Comment:
Heads up: be careful with your merge, I think this import is needed for recent already merged changes to `get_dashboard_matches`